alpha/experience-platform-extension/auto-collected-data.md

```diff
@@ -0,0 +1,70 @@
+# Automatically collected information
+
+The Experience Platform Mobile extension automatically collects and adds information to each Platform event that is sent to the Adobe Experience Platform. The following sections provide details about the collected information:
+
+## Identity
+
+The [IdentityMap](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/identitymap.schema.md) defines a map that contains end-user identities.
+
+#### ECID
+
+The Experience Cloud Identifier (ECID)  that is generated by the Adobe Experiece Cloud Mobile SDK is added to the Platform event's `IdentityMap` . If the event contains an `IdentityMap` object, the ECID is added to the `IdentityMap` If the event does not contain the object, a new object is created and is added to the Platform event.
+
+------
+
+
+
+## Device
+
+[Device](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/device.schema.md) is included in the [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and details data about the mobile device. The following are the device properties the Experience Platform Mobile extension automatically adds to each Platform event.
```

Comment:
Please reword the last sentencer as follows:
"Here are the device properties the Experience Platform Mobile extension automatically adds to each Platform event:" --> please replace the period with a colon.